src/framex/utils/cache.py

```
@@ -0,0 +1,80 @@
+        if isinstance(item, dict):
+            if item.get("__type__") == "dynamic_obj":
+                try:
+                    module = importlib.import_module(item["__module__"])
+                    cls = getattr(module, item["__class__"])
+
+                    cleaned_data = {k: restore_models(v) for k, v in item["data"].items()}
+
+                    if hasattr(cls, "model_validate"):
+                        return cls.model_validate(cleaned_data)
+                    return cls(**cleaned_data)
+                except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)